cbmc.inc

```diff
@@ -29,11 +29,11 @@ ulimit -v 15000000 ; \
 EC=42 ; \
 for c in 2 6 12 17 21 40 200 400 1025 2049 268435456 ; do \
 echo "Unwind: $c" > $LOG.latest ; \
-./cbmc-binary --graphml-witness $LOG.witness --slice-formula --unwind $c --stop-on-fail --object-bits $OBJ_BITS $PROPERTY $LOG.bin >> $LOG.latest 2>&1 ; \
+./cbmc-binary --no-unwinding-assertions --graphml-witness $LOG.witness --slice-formula --unwind $c --stop-on-fail --object-bits $OBJ_BITS $PROPERTY $LOG.bin >> $LOG.latest 2>&1 ; \
```

Comment:
@paolo-di-biase, there are two additional changes that need to be applied for CBMC 6.x:
- Add `--no-standard-checks` to all `cbmc-binary` invocations
- Remove `--object-bits` from all `goto-cc` invocations (but not from `cbmc-binary` invocations)
The latter should fix the errors on `c/weaver` benchmarks.